tests/sentry/incidents/test_tasks.py

```diff
@@ -49,6 +49,13 @@ def test_simple(self):
         send_subscriber_notifications(activity.id)
         self.send_async.assert_called_once_with([user.email])
 
+    def test_invalid_types(self):
+        for activity_type in (IncidentActivityType.CREATED, IncidentActivityType.DETECTED):
```

Comment:
Is `detected` here because it doesn't exist yet?

Reply:
Detected is an alert rather than a notification, so separate to this since it can't rely on subscriptions.

As a side note, I think that having created/detected as statuses was a mistake. It'll make things awkward if we want to re-open an issue, etc. So if I get time I plan on having status just be `open/closed` and maybe `reopened`, and having a `type` field for manual/automatic.